xAPI.md

```diff
@@ -1884,6 +1884,8 @@ Complete IRI validation is extremely difficult, so much of the burden for ensuri
 * A library SHOULD be used to construct IRIs, as opposed to string concatenation. 
 * Values SHOULD be considered to be case sensitive unless specified otherwise. 
 * Lowercase SHOULD be used to send case insensitive data.
+* Additional properties SHOULD* NOT be added to statements and other objects unless explicitly allowed and the 
+LRS MAY* reject statements and objects containing such additional properties.
```

Comment:
It might be good to add a note to APs that there are definitely LRSs that do so. The other objects bit is a bit weird since it isn't possible to set any other objects (that have a spec-restricted property list) with the spec -- about and person objects are only returned by the LRS.